src/Microsoft.ML.StandardTrainers/Standard/MulticlassClassification/PairwiseCouplingTrainer.cs

```diff
@@ -40,8 +40,8 @@ namespace Microsoft.ML.Trainers
     /// |  |  |
     /// | -- | -- |
     /// | Machine learning task | Multiclass classification |
-    /// | Is normalization required? | Depends on underline binary classifier |
-    /// | Is caching required? | Depends on underline binary classifier |
+    /// | Is normalization required? | Depends on the underlying binary classifier |
+    /// | Is caching required? | Yes (it makes many passes over training data) |
```

Comment:
>required [](start = 21, length = 8)

I would question word required, but I guess it's too late. It's not like it wont work without caching, its more like caching would be beneficial for training.
Also not sure why we need to have explanation in braces.

Reply:
In hindsight, we should have said "can benefit from caching?" instead. 

Removed the explanation. 

---
In reply to: [277470911](https://github.com/dotnet/machinelearning/pull/3516#discussion_r277470911) [](ancestors = 277470911)